applications/DEMApplication/python_scripts/DEM_material_test_script.py

```diff
@@ -334,7 +338,7 @@ def MeasureForcesAndPressure(self):
 
         dt = self.spheres_model_part.ProcessInfo.GetValue(DELTA_TIME)
 
-        self.strain += -100*self.length_correction_factor*1.0*self.LoadingVelocity*dt/self.height
+        self.strain += -100*self.length_correction_factor*1.0*self.LoadingVelocity*dt/self.height # Here the 2.0 is not included, there's a 1.0! TODO
```

Comment:
we need to clarify this TODO on the phone

Reply:
We concluded that this file must never use the value of  `self.LoadingVelocity` , but deduct it from the boundary conditions, otherwise, they could be inconsistent.